packages/camera/camera_windows/windows/test/capture_controller_test.cpp

```diff
@@ -337,6 +383,35 @@ TEST(CaptureController, ReportsInitializedErrorEvent) {
   engine = nullptr;
 }
 
+TEST(CaptureController, ReportsInitializedAccessDeniedEvent) {
+  ComPtr<MockCaptureEngine> engine = new MockCaptureEngine();
+  std::unique_ptr<MockCamera> camera =
+      std::make_unique<MockCamera>(MOCK_DEVICE_ID);
+  std::unique_ptr<CaptureControllerImpl> capture_controller =
+      std::make_unique<CaptureControllerImpl>(camera.get());
+  std::unique_ptr<MockTextureRegistrar> texture_registrar =
+      std::make_unique<MockTextureRegistrar>();
+
+  int64_t mock_texture_id = 1234;
```

Comment:
uber-pendantic-nit: coust be const/constexpr.

Reply:
Fixed!